mathics/builtin/files_io/files.py

```diff
@@ -384,59 +377,21 @@ class Get(PrefixOperator):
     precedence = 720
     summary_text = "read in a file and evaluate commands in it"
 
-    def eval(self, path, evaluation: Evaluation, options: dict):
+    def eval(self, path: String, evaluation: Evaluation, options: dict):
         "Get[path_String, OptionsPattern[Get]]"
 
-        def check_options(options):
-            # Options
-            # TODO Proper error messages
-
-            result = {}
-            trace_get = evaluation.parse("Settings`$TraceGet")
-            if (
-                options["System`Trace"].to_python()
-                or trace_get.evaluate(evaluation) is SymbolTrue
-            ):
-                import builtins
-
-                result["TraceFn"] = builtins.print
-            else:
-                result["TraceFn"] = None
-
-            return result
+        trace_fn = None
+        trace_get = evaluation.parse("Settings`$TraceGet")
+        if (
+            options["System`Trace"].to_python()
+            or trace_get.evaluate(evaluation) is SymbolTrue
+        ):
+            trace_fn = builtins.print
```

Comment:
This code now only handles option processing. The rest of the code is in `mathics.eval.files_io.files`